monitoring/monitorlib/messagesigning/message_signer.py

```diff
@@ -0,0 +1,104 @@
+import base64
+import time
+from Crypto.PublicKey import RSA
+from Crypto.Signature import pkcs1_15
+from Crypto.Hash import SHA256
+from loguru import logger
+import urllib.parse
+from monitoring.monitorlib.messagesigning.hasher import get_content_digest
+from monitoring.monitorlib.messagesigning.uft import get_x_utm_jws_header
+
+def get_signed_headers(object_to_sign, private_key_path, cert_url):
+    signed_type = str(type(object_to_sign))
+    is_signing_request = "PreparedRequest" in signed_type
+    sig, sig_input = _get_signature(object_to_sign, signed_type, private_key_path)
+    content_digest = (
+        get_content_digest(object_to_sign.body)
+        if is_signing_request
+        else get_content_digest(object_to_sign.get_data())
+    )
+    signed_headers = {
+        "x-utm-message-signature-input": "utm-message-signature={}".format(sig_input),
+        "x-utm-message-signature": "utm-message-signature=:{}:".format(sig),
+        "x-utm-jws-header": get_x_utm_jws_header(cert_url),
```

Comment:
Seems like this should be in uft.py since x-utm-jws-header isn't specified by IETF.

Basically, whenever I see "x-utm", I would expect that is a UFT-specific thing that shouldn't be in a generic message signing library -- the generic/IETF thing would accept some kind of parameter/argument, and the UFT-specific library would supply "x-utm*" for those arguments.